Dockerfile

```diff
@@ -252,6 +252,7 @@ RUN perl -pi -e 's/multi-user.target/docker-entrypoint.target/g' /usr/local/lib/
   mkdir -p /etc/bash_completion.d && \
   nerdctl completion bash >/etc/bash_completion.d/nerdctl && \
   mkdir -p -m 0755 /etc/cni
+RUN sudo sysctl -w net.ipv6.conf.all.forwarding=1 && sudo sysctl -w net.ipv4.ip_forward=1
```

Comment:
The applied configuration disappears after the `RUN` line, so you have to specify this via `docker run --sysctl`